framework/application/modules/Session/Session.php

```diff
@@ -364,32 +395,6 @@ public function setPostLogin($post_login=REDIRECT_AFTER_LOGIN)
 	/*** private methods ***/
 
 	/**
-	 * setMessage
```

Comment:
I moved this back up into mutators. I bfeel that mutators should be grouped for ease, no matter what their access is. private, public, and protected areas are great for all other method in a class.